bolt/include/bolt/Core/BinaryFunction.h

```diff
@@ -2094,6 +2103,10 @@ class BinaryFunction {
   // adjustments.
   void handleAArch64IndirectCall(MCInst &Instruction, const uint64_t Offset);
 
+  /// Processes code section to identify function references.
+  void processInstructionsForFuncReferences(BinaryContext &BC,
+                                            const MCInst &Inst);
```

Comment:
nit: make it explicit that we check address-taken references? 
Please also update the comment